src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java

```diff
@@ -373,6 +371,14 @@ public void removeNode(String nodeName) {
         return (T)internal.nodes.getItem(nodeName).getExternalNode();
     }
 
+    public void beginTransaction(){
+        internal.beginTransaction();
+    }
+
+    public void endTransaction(){
```

Comment:
How about using commitTransaction(), this is what is often used in SQL transactions, so developers might feel more confident about what the method does.

We should add the rollbackTransaction() method which does nothing other than clearing the attribute list...